uber/templates/base.html

```diff
@@ -93,7 +93,13 @@
             if (message.length) {
                 toastr.info(message);
             }
```

Comment:
https://stackoverflow.com/questions/25207147/datatable-vs-datatable-why-is-there-a-difference-and-how-do-i-make-them-w